src/plugins/ui_stateful/edition.ts

```diff
@@ -201,10 +201,11 @@ export class EditionPlugin extends UIPlugin {
           this.resetContent();
         }
         if (cmd.sheetIdFrom !== cmd.sheetIdTo) {
+          const activePositionInSheetTo = this.getters.getSheetPosition(cmd.sheetIdTo);
```

Comment:
the sheetIdTo is actually the active sheet, because it was already updated in GridSelectionPlugin. You could write 
```suggestion
          const activePosition = this.getters.getActivePosition();
```